test/pylib/harness_cli.py

```diff
@@ -125,3 +125,10 @@ async def nodes(self) -> List[str]:
         host_list = await self._request_and_check("http://localhost/cluster/nodes",
                                                   "Error getting list of nodes")
         return host_list.split(',')
+
+    async def mark_dirty(self) -> None:
+        """Manually mark current cluster dirty.
+           To be used when a server was modified outside of this API."""
+        self.dirty = True
+        await self._request_and_check("http://localhost/cluster/mark-dirty",
```

Comment:
Again, why "localhost"? Shouldn't this be the Unix-domain socket filename?

Reply:
As before, `aiohttp`.